app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -233,6 +240,13 @@ private void populateV1APIMeterReadingEntry(JSONArray array, Calibration record)
             json.put("date", record.timestamp);
             json.put("dateString", format.format(record.timestamp));
             json.put("mbg", record.bg);
+            json.put("xDrip_slope", record.slope);
+            json.put("xDrip_intercept", record.intercept);
+            json.put("xDrip_estimate_raw_at_time_of_calibration", record.estimate_raw_at_time_of_calibration);
+            json.put("xDrip_slope_confidence", record.slope_confidence);
+            json.put("xDrip_sensor_confidence", record.sensor_confidence);
+            json.put("xDrip_raw_timestamp", record.raw_timestamp);
+            json.put("sysTime", format.format(record.timestamp));
```

Comment:
The data that starts with xDrip is the data that is needed for xDrip to work correctly, in more details, this are needed for the calibration graph and and calibration table.

As for putting this here or on the calibration. Here are my thoughts about this:
We would like the NightwatchPro to work with NS data and also with the xDrip mode. Obviously people that work with xDrip will have more data (like calibration graph and so on).

NS has divided the data that xDrip calls calibration into two places, one is a meter record and one is the calibration itself. Both by the way, don't contain the exact fields that xDrip uses for calibration. This is why I need extra fields.
So, why did I put it here? well the most important data in a calibration (that is displaying it on the screen) is the time and the actual value measured. 
So, reading this "meter record" will allow me to display the data for xDrip users as well as for NS users.